packages/language-php/grammars/tree-sitter/queries/highlights.scm

```diff
@@ -233,19 +233,56 @@
 
 (class_declaration (name) @entity.name.type.class.php)
 
+(class_declaration (base_clause (name) @entity.other.inherited-class.php))
+(class_declaration (class_interface_clause (name) @entity.other.implemented-interface.php))
```

Comment:
These two lines  were removed during the recent rebase, but came back in here. I can remove them, or resolve the conflict if/when they're readded. Or of course remove them if that was intentional.